tests/phpunit/QueryPrinterRegistryTestCase.php

```diff
@@ -82,6 +82,6 @@ static function ( array $args ) use ( $phpFails ) {
 	public function testConstructor( $format, $isInline ) {
 		$instance = $this->newInstance( $format, $isInline );
 
-		$this->assertInstanceOf( '\SMW\ResultPrinter', $instance );
+		$this->assertInstanceOf( '\SMW\Query\ResultPrinters\ResultPrinter', $instance );
```

Comment:
Please just import it, and use `ResultPrinter` in the comments and `ResultPrinter::class` in this assertion

Reply:
Thanks, done.